Some code warnings for test programs

```
tst_bitgroom.c: In function ‘main’:
tst_bitgroom.c:128:16: warning: unused variable ‘y’ [-Wunused-variable]
  128 |         int x, y, f;
      |                ^

tst_zstandard.c: In function ‘main’:
tst_zstandard.c:122:16: warning: unused variable ‘y’ [-Wunused-variable]
  122 |         int x, y, f;
      |                ^
```

--- a/configure.ac
+++ b/configure.ac
@@ -408,4 +408,5 @@ AC_CONFIG_FILES([Makefile
         ])
 AC_OUTPUT()
 
+# Print build summary.
 cat libccr.settings

--- a/test/tst_bitgroom.c
+++ b/test/tst_bitgroom.c
@@ -125,7 +125,7 @@ main()
         int varid;
         int *data_out;
         int *data_in;
-        int x, y, f;
+        int x, f;
         int nsd_in, bitgroom;
 
         if (!(data_out = malloc(NX_BIG * NY_BIG * sizeof(int)))) ERR;

--- a/test/tst_zstandard.c
+++ b/test/tst_zstandard.c
@@ -119,7 +119,7 @@ main()
         int varid;
         int *data_out;
         int *data_in;
-        int x, y, f;
+        int x, f;
         int level_in, zstandard;
 
         if (!(data_out = malloc(NX_BIG * NY_BIG * sizeof(int)))) ERR;

